src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.Common.targets

```diff
@@ -33,8 +33,32 @@ Copyright (c) .NET Foundation. All rights reserved.
   <UsingTask TaskName="NETSdkError" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
   
   <!--
+    ============================================================
+                                        ShouldQueryForProperties
+
+    Indicate that this project cross-targets, so before building
+    it, a referring project must call GetReferenceProperties
+    to get the matching TargetFramework to specify.
+  -->
+  <Target Name="ShouldQueryForProperties"
+          Returns="$(_ReferringProjectShouldCallGetReferenceProperties)">
+    <PropertyGroup>
+      <!-- indicate to caller that project is RID agnostic so that a global property RuntimeIdentifier value can be removed -->
+      <_IsRidAgnostic>false</_IsRidAgnostic>
+      <_IsRidAgnostic Condition=" '$(RuntimeIdentifier)' == '' and '$(RuntimeIdentifiers)' == '' ">true</_IsRidAgnostic>
```

Comment:
I believe this will not deal well with the case where RuntimeIdentifier is specified as a global property (-r from CLI). We removed it from global properties for GetTargetFrameworkProperties, but not from ShouldQueryForProperties :(